admin-cli/executor/client.go

```diff
@@ -67,3 +69,46 @@ func NewClient(writer io.Writer, metaAddrs []string) *Client {
 		Perf:   aggregate.NewPerfClient(metaAddrs),
 	}
 }
+
+// NewClientWithoutExit creates a client for accessing Pegasus cluster for use of admin-cli.
+// This function will not call os.Exit.
+func NewClientWithoutExit(writer io.Writer, metaAddrs []string) (*Client, error) {
```

Comment:
This fucntion is similar to the function above, how about unify them?

Reply:
Thinks, i will unify them.